core/src/test/java/org/javarosa/core/util/test/MathUtilsTest.java

```diff
@@ -0,0 +1,27 @@
+package org.javarosa.core.util.test;
+
+import org.javarosa.core.util.MathUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.lang.Math;
+
+/**
+ * @author ftong
+ */
+public class MathUtilsTest {
+    /**
+     * Test that standard and J2ME asins behave similarly.
+     * More extensive tests are unnecessary because the code was adapted from
+     * http://www.netlib.org/fdlibm/e_asin.c, as cited in the function comments.
+     */
+    @Test
+    public void testAsinsBehaveSimilarly() {
+        double tolerance = 0.00000001;
```

Comment:
This tolerance is arbitrary. When I printed out Math.asin(1) and MathUtils.asin(1), they were completely identical.